practicesharp/Calculator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+
+    
+
+
+    class Calculator
+    {
+        public double Calculate(string equation)
+        {
+            Number number= new Number(ref equation);
```

Comment:
Using "ref" is a bad practice here, you don't expect a constructor to change the objects it receives.
Even if it's a function, better just "return" the new value.